### DISCUSSION TITLE
Sourcemap support for _worker.js file when running with cloudflare

### DISCUSSION THREAD

**Discussion by TorbjornHoltmon** [withastro/roadmap#535] [Proposal]
# Sourcemap support for server files / _worker.js file

We have been using sentry to log server error stack traces in Cloudflare workers. But when using Astro, I cannot find any option to output source map for the _worker.js file.

Source maps are needed for sentry to be able to give us stack traces that make sense since everything is minified.

**Comment by lucaswalter** [comment]
This would be helpful for me as well.
